applications/serial_lte_modem/doc/slm_data_mode.rst

```diff
@@ -54,6 +54,8 @@ To exit data mode, the MCU sends the termination command set by the ``CONFIG_SLM
 
 When instructed to exit data mode, the SLM application returns the AT command response ``OK``.
 
+The SLM application exits data mode if current sending function fails, in which case it returns the AT command response ``ERROR``.
```

Comment:
If the current sending function fails, the SLM application exits data mode and returns the AT command response ``ERROR``.

Reply:
Applied, thanks